src/util-buffer.c

```diff
@@ -48,8 +48,7 @@ MemBuffer *MemBufferCreateNew(uint32_t size)
         return NULL;
     }
     buffer->size = size;
-    buffer->buffer = (uint8_t *)buffer + sizeof(MemBuffer);
-
+    MemBufferReset(buffer);
```

Comment:
a simpler fix would have been to use SCCalloc above?

Reply:
I think it'd be equivalent. I like the explicit reset (and it's an inline function).

Your call though -- which would you prefer?

Reply:
Prefer less code, so SCCalloc.

Would consider it a nit, but now that we need another round because of the SV test issue, I prefer this to be addressed as well.